docs/UserGuide.adoc

```diff
@@ -462,6 +467,7 @@ Example: +
 After running a match command on a specific patient with `match ic/**NRIC**`, you will get the list of donors that are compatible with said patient.
 You can then run `sort rate` to obtain the list of donors with the highest success rate at the top of the list.  +
 
+// tag::process[]
```

Comment:
Just a note, if you want to include a tag like this: `// tag::process[]`, you need to include the end tag after the section ends. 

The end tag is this: `// end::process[]`

Reply:
Added.